pages/common/lldb.md

```diff
@@ -0,0 +1,15 @@
+# lldb
+
+> The LLDB Debugger.
+
+- Debug an executable:
+
+`lldb {{executable}}`
+
+- Attach lldb to running process with id {{PID}}:
+
+`lldb -p {{PID}}`
+
+- Wait for process to launch and attach to it:
+
+`lldb -w -n {{name}}`
```

Comment:
Lets change name to process_name to clarify things better. Also, a question - what if there are 2 processes with the same name. If I use the command without the `-w` flag, which process will it attach to ?